service/src/main/kotlin/fi/espoo/evaka/jamix/JamixService.kt

```diff
@@ -92,38 +105,99 @@ class JamixService(
 
     fun syncDiets(db: Database.Connection, clock: EvakaClock, job: AsyncJob.SyncJamixDiets) {
         if (client == null) error("Cannot sync diet list: JamixEnv is not configured")
-        fetchAndUpdateJamixDiets(client, db)
+        val warningEmails =
+            fetchAndUpdateJamixDiets(client, db, clock, emailEnv.sender(Language.fi))
         fetchAndUpdateJamixTextures(client, db)
+        warningEmails.forEach { emailClient.send(it) }
```

Comment:
This doesn't look transactionally safe. If for example `fetchAndUpdateJamixTextures` fails then the whole job is run again, but the diet deletion has already been committed, so emails won't be sent on a retry either. Also need to be careful of not sending already sent emails again. I believe the correct way is to plan email sending async jobs as part of the same transaction that nulls the diets.